lib/svn2git/migration.rb

```diff
@@ -154,6 +154,7 @@ def parse(args)
 
     def self.escape_quotes(str)
       str.gsub(/'|"/) { |c| "\\#{c}" }
+         .gsub(/\$/)  { |c| '\\$' }
```

Comment:
Would [`Shellwords.escape`](http://ruby-doc.org/stdlib-1.9.3/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellescape) be a good fit here?